src/db_controller.js

```diff
@@ -1,10 +1,6 @@
 const mongoose = require("mongoose");
 
-// Default mongo port
-const DB_PORT = 27017;
-const DB_HOST = process.env.DB_HOSTNAME || "localhost";
-
-const MONGO_URI = process.env.MONGO_URI || `mongodb://${DB_HOST}:${DB_PORT}`;
+const MONGO_URI = process.env.MONGO_URI || "mongodb://localhost:27017";
```

Comment:
Should we maybe not have a default at all and just blow up when there is none specified (I'd think trying to connect to an undefined URI probably will do the trick, but adding a simple condition here might also work).